src/terra-dev-site/contributing/Conventions.d.contributing.md

```diff
@@ -13,8 +15,8 @@
+<div className="tui-illustration-grid-col tui-illustration-grid-col-illustration">
+<PairProgramingIcon />
```

Comment:
Looks like this icon may not be rendering correctly in the PR deployment.

<img width="1317" alt="Screen Shot 2019-10-01 at 9 22 33 AM" src="https://user-images.githubusercontent.com/633148/65970941-0892a400-e42d-11e9-9174-909e172b2fc3.png">

Reply:
It was a spelling issue, I didn't spell the component the same as the import. I'm a bit surprised MDX didn't throw a render error.